### ISSUE TITLE
Changes in nunjuck files in templates does not show on browser refresh

### ISSUE THREAD

**Issue by thomas-kahn** [DefaultSimon/gulp-nunjucks-sass-template#3]
In the gulpfile the build is told to watch the nunjuck files in the templates folder:
```
function watchHtml() {
  return gulp.watch([`${srcBase}pages/**/*.njk`, `${srcBase}templates/**/*.njk`], html);
}
```
...but as far as I can tell changes in the files in templates are not reflected in the browser when you save the file. Njunjucks files in the pages folder on the other hand seem to work fine and changes are reflected in the browser on save.

**Comment by simongoricar** [comment]
Thank you for raising this issue, it's solved as of the latest commit e401607ed1982cf8bda98b06848c9f5380c596b4 😃

**Comment by thomas-kahn** [comment]
Thanks for a swift fix! You're doing a great job!
